packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -947,7 +947,7 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
       final Element elementB = find.text('b').evaluate().first;
```

Comment:
can you file a bug on flutter/flutter to reduce the redundancy in his test file? And link to that issue from the top of this file? thanks

Reply:
Issue is already created, good call to add a todo here!